src/index.html

```diff
@@ -9,10 +9,18 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
+    <script
+      type="module"
+      src="./scripts/main.js"
+    ></script>
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div
+        class="calendar calendar--start-day-sun calendar--month-length-31"
+      ></div>
```

Comment:
It's a good practice to avoid using too many classes on a single element. Consider using BEM methodology or another CSS methodology to manage your classes in a more scalable way.